src/lib/firebase.js

```diff
@@ -17,4 +17,4 @@ var firebaseConfig = {
 
 let fb = firebase.initializeApp(firebaseConfig);
 
-export { fb };
+export default fb;
```

Comment:
Thank you for that! I only added it because it was yelling at me that it wanted a default export and I had it imported as fb so I was calling fb.fb lol just needed better naming. Can totally fix that.

Reply:
Fixed!